lib/recurly/hosted-fields.js

```diff
@@ -140,4 +175,24 @@ export class HostedFields extends Emitter {
       recurly: this.config.recurly
     };
   }
+
+  serveAsSingleField() {
+    const fields = this.config.recurly.fields;
+    const elementCounts = {
+      'notFound': 0,
+      'found': 0
+    }
+    FIELD_TYPES.forEach((type, _) => {
+      const selector = fields[type].selector;
+      const element = dom.element(global.document.querySelector(selector));
+      element ? elementCounts['found']++ : elementCounts['notFound']++;
+    });
+    /* Verify that at least one element is NOT in the dom. If at least one element
+     * is not in the dom, then we will configure a single hosted field. */
+    return elementCounts['notFound'] > 1;
+  }
```

Comment:
I don't think this method is necessary, and I don't think single fields need to be treated differently than others. No matter if there are one or many hosted fields being configured, each should get a hidden tabbing sink, no?